Gemfile.lock

```diff
@@ -1,55 +1,55 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (6.0.3.2)
-      actionview (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actionpack (6.0.3.3)
+      actionview (= 6.0.3.3)
+      activesupport (= 6.0.3.3)
       rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actionview (6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actionview (6.0.3.3)
+      activesupport (= 6.0.3.3)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activemodel (6.0.3.2)
-      activesupport (= 6.0.3.2)
-    activerecord (6.0.3.2)
-      activemodel (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
-    activesupport (6.0.3.2)
+    activemodel (6.0.3.3)
+      activesupport (= 6.0.3.3)
+    activerecord (6.0.3.3)
+      activemodel (= 6.0.3.3)
+      activesupport (= 6.0.3.3)
+    activesupport (6.0.3.3)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
       zeitwerk (~> 2.2, >= 2.2.2)
     builder (3.2.4)
-    concurrent-ruby (1.1.6)
+    concurrent-ruby (1.1.7)
     crass (1.0.6)
     double_entry (1.0.1)
       activerecord (>= 3.2.0)
       activesupport (>= 3.2.0)
       money (>= 6.0.0)
       railties (>= 3.2.0)
     erubi (1.9.0)
-    i18n (1.8.3)
+    i18n (1.8.5)
       concurrent-ruby (~> 1.0)
-    json (2.3.0)
-    jwt (2.2.1)
-    jwt_signed_request (2.5.4)
+    json (2.3.1)
+    jwt (2.2.2)
+    jwt_signed_request (2.6.0)
       jwt (>= 1.5.0)
       rack
-    loofah (2.6.0)
+    loofah (2.7.0)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
     method_source (1.0.0)
     mini_portile2 (2.4.0)
-    minitest (5.14.1)
-    money (6.13.7)
+    minitest (5.14.2)
+    money (6.13.8)
```

Comment:
### [money](https://rubymoney.github.io/money)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 6.13.7 → 6.13.8

[_[change-log](https://github.com/RubyMoney/money/blob/master/CHANGELOG.md), [source-code](https://github.com/RubyMoney/money/)_]





<details>
<summary>Commits</summary>

A change of **12** commits. See the full changes on [the compare page](https://github.com/RubyMoney/money/compare/v6.13.7...v6.13.8).

These are the first 10 commits:
- (5f1847d) [Add Ruby 2.7.0 to TravisCI (#907)](https://github.com/RubyMoney/money/commit/5f1847d01853e73ea69848b85d9fa468594a7dc0)
- (3f265a0) [Update symbol for XOF (#908)](https://github.com/RubyMoney/money/commit/3f265a0ad5c03a55bbc229a0d0746e2463d3f7cd)
- (7b01281) [Update UYU currency symbol (#916)](https://github.com/RubyMoney/money/commit/7b01281a6606359ef8b5e1e5b48e0673d6941c60)
- (6b8fbf0) [Allow double conversion using same bank (#918)](https://github.com/RubyMoney/money/commit/6b8fbf07158e162fb5a7d5be88c20247e900de38)
- (a6bc544) [Document how to localize currency position format (#919)](https://github.com/RubyMoney/money/commit/a6bc5441ca1225481976243c8e93d70f8b1667c3)
- (f4d3f6c) [Prevent method redefinition warnings (#920)](https://github.com/RubyMoney/money/commit/f4d3f6c3df4938b714a0b5f1986b45c39263d44c)
- (69a20c7) [Suppress warnings when running specs (#922)](https://github.com/RubyMoney/money/commit/69a20c728d280f6ba8cc75d05d378ea09bf25313)
- (e6aa20e) [Fix yard documentation (#923)](https://github.com/RubyMoney/money/commit/e6aa20e88d873e1ffc7de7f3d705b2e25417efea)
- (2efd45f) [Print warning when using unsafe serializer for rate import (…](https://github.com/RubyMoney/money/commit/2efd45f45062e75402c795d8a71c2206811e65db)
- (265de50) [Move Icelandic symbol after the amount (#929)](https://github.com/RubyMoney/money/commit/265de50b987b8c69261dde013c8dd18a567279df)


</details>